docs/Account.md

```diff
@@ -435,6 +439,27 @@ response_len: felt
 response: felt*
 ```
 
+### `_unsafe_execute`
+
+Is the internal method that performs the following task inside the `execute` and `eth_execute` functions:
```

Comment:
```suggestion
It's an internal method that performs the following tasks:
```
`_unsafe_execute` is not aware of who's calling it, it may be used by other `execute` flavors too.